src/components/About/OurStory.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+import '../../styles/OurStory.css';
+import Blurbs from "./Blurbs.js";
+
+class OurStory extends Component {
+  render() {
+    return (
+      <div className="OurStory" id="OurStory">
+        <Blurbs />
+      </div>
+    );
+  }
+}
+
+export default OurStory;
```

Comment:
About/OurStory.js: Code is clean and creating OurStory.js improves organization of components for future additions/amendments.